doc/examples/mocha/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "devDependencies": {
     "axe-core": "^4.6.2",
+    "chai": "^4.4.0",
```

Comment:
Without the pin `karma-chai` installs the latest version of `chai` which would install v5.

Reply:
Pinning it to solve that problem is reasonable, but can we keep the version used here and the version used by the root package/lockfile consistent?